actionview/lib/action_view/helpers/form_options_helper.rb

```diff
@@ -597,15 +597,19 @@ def time_zone_options_for_select(selected = nil, priority_zones = nil, model = :
       # Returns a string of option tags for the days of the week.
       #
       # Options:
-      # * <tt>:index_as_value</tt> - Defaults to false, set to true to use the index of the weekday as the value.
+      # * <tt>:index_as_value</tt> - Defaults to false, set to true to use the indexes from
+      # `I18n.translate("date.day_names")` as the values. By default, Sunday is always 0.
       # * <tt>:day_format</tt> - The I18n key of the array to use for the weekday options.
       # Defaults to :day_names, set to :abbr_day_names for abbreviations.
+      # * <tt>:beginning_of_week</tt> - Defaults to Date.beginning_of_week, accepts
+      # :sunday, :monday, or :saturday values to set the first day on the list.
```

Comment:
We support all days now, and can just omit these specific references:

```suggestion
      # * <tt>:beginning_of_week</tt> - Defaults to Date.beginning_of_week.
```

Reply:
Good call